test/ui/RelevanceInspector/AgGridTest.ts

```diff
@@ -0,0 +1,53 @@
+import { loadAgGridLibrary, reset } from '../../../src/ui/RelevanceInspector/AgGrid';
+import { $$ } from '../../Test';
+declare global {
+  interface Window {
+    [propName: string]: any;
+  }
+}
+
+let oldAgGridReference: any;
+
+export function AgGridTest() {
+  describe('AgGrid', () => {
+    let doc: HTMLDocument;
+
+    beforeEach(() => {
+      reset();
+      if (window.agGrid) {
+        oldAgGridReference = window.agGrid;
+        delete window.agGrid;
+      }
+      doc = document.implementation.createHTMLDocument('testing');
+    });
+
+    afterEach(() => {
+      window.agGrid = oldAgGridReference;
```

Comment:
I want to test that the "AgGrid" can be loaded, which is why i need to "remove" it for each of those test in this suite.

But for the other tests for the relevance inspector, I need to ag grid library to be available so that the component can actually work and be tested, which is why I put it back after each test in this suite.